src/MakeListingDisplay/components/MakeListingDisplay.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+import InputField from './InputField';
+
+import '../styles/MakeListingDisplay.css';
+
+/**Component class that displays the input fields for a user to post a listing */
+class MakeListingDisplay extends Component {
+    render(){
+        return (
+            <div>
+                <div id="header">
+                    <h1>Post a Listing</h1>
+                </div>
+                <form action="/listings" id="listingForm" method="POST">
+                    <InputField fieldHeader="Title:" fieldName="title" fieldType="text" />
+                    <InputField fieldHeader="Description:" fieldName="description" fieldType="text" />
+                    <InputField fieldHeader="Total number of rooms in apartment:" fieldName="numRooms" fieldType="number" />
+                    
+                    <p>Number of rooms for rent:</p>
+                    <div id="singleInfo">
```

Comment:
For now, I have added it so that the sharedinfo inputs and singleinfo inputs default to 0, so if a user only has one of those 2 options, they can just leave it as is